src/picongpu/include/traits/AdiosToPIC.tpp

```diff
@@ -44,6 +44,12 @@ namespace traits
         typedef unsigned int type;
     };
 
+    template<>
+    struct AdiosToPIC<adios_unsigned_long>
+    {
+        typedef uint64_t type;
```

Comment:
We would not be the only ones doing that: https://github.com/Kitware/VTK/blob/master/IO/ADIOS/ADIOSScalar.cxx#L64, http://fossies.org/linux/VTK/IO/ADIOS/ADIOSUtilities.cxx